src/librustdoc/html/render/mod.rs

```diff
@@ -2477,7 +2492,15 @@ fn sidebar_module(buf: &mut Buffer, items: &[clean::Item]) {
     }
 
     if !sidebar.is_empty() {
-        write!(buf, "<div class=\"block items\"><ul>{}</ul></div>", sidebar);
+        write!(
+            buf,
+            "<section class=\"items\">\
+                 <div class=\"block\">\
```

Comment:
I was thinking that `<section>` (no class) would _replace_ `<div class=\"block\">`. Then the relevant CSS selectors would be `.sidebar section`.

Reply:
Oh I see. Even better then!